tutorials/ComplexArithmetic/ComplexArithmetic.ipynb

```diff
@@ -0,0 +1,623 @@
+    ">\n",
+    "<br/>\n",
+    "<details>\n",
+    "    <summary><strong>Need a hint? Click here</strong></summary>\n",
+    "    In particular, you might be interested in <a href=https://docs.python.org/3/library/math.html#math.sqrt>Python's square root function</a>\n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@exercise\n",
+    "def modulus(x : Complex) -> float:\n",
+    "    return # ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imaginary Exponents\n",
+    "\n",
+    "The next complex operation we're going to need is exponentiation. Raising an imaginary number to an integer power is a fairly simple task, but raising something to an imaginary power, or raising an imaginary (or complex) number to a real power isn't quite as simple.\n",
+    "\n",
+    "Let's start with raising real numbers to imaginary powers. Specifically, let's start with a rather special real number - Euler's constant, $e$:\n",
+    "\n",
+    "$$e^{ix} = \\cos x + i\\sin x$$\n",
+    "\n",
+    "Where $x$ is in radians.\n",
+    "\n",
+    "Explaining why that happens is somewhat beyond the scope of this tutorial, as it requires some calculus, so we won't do that here. If you are curious, you can see [this video](https://youtu.be/v0YEaeIClKY) for a beautiful intuitive explanation, or [the Wikipedia article](https://en.wikipedia.org/wiki/Euler%27s_formula#Proofs) for a more mathematically rigorous proof.\n",
+    "\n",
+    "Here are some examples of this formula in action:\n",
+    "\n",
+    "$$e^{i\\pi/4} = \\frac{1}{\\sqrt{2}} + \\frac{i}{\\sqrt{2}} \\\\\n",
+    "e^{i\\pi/2} = i \\\\\n",
+    "e^{i\\pi} = -1 \\\\\n",
+    "e^{2i\\pi} = 1$$\n",
+    "\n",
+    "> One interesting consequence of this is Euler's Identity:\n",
+    ">\n",
+    "> $$e^{i\\pi} + 1 = 0$$\n",
+    ">\n",
+    "> While this doesn't have any notable uses, it is still an interesting identity to consider, as it combines 5 fundamental constants of algebra into one expression.\n",
+    "\n",
+    "We can also calculate complex powers of $e$ as follows:\n",
+    "\n",
+    "$$e^{a + bi} = e^a \\cdot e^{bi}$$\n",
+    "\n",
+    "Finally, using logarithms to convert the base of the exponent, we can use this to find complex powers of any positive real number.\n",
+    "\n",
+    "That being said, let's return to $e^{ix}$ for a moment. If we map $\\cos x + i\\sin x$ onto the complex plane, we can see that it lands on a **unit circle** around $0 + 0i$. This means that its modulus is always $1$. You can also verify this algebraically: $\\cos^2 x + \\sin^2 x = 1$. This property will be very useful in the next section."
```

Comment:
I would move this sentence to the beginning of the section "Polar coordinates" - same as in previous cases, this is important later but at this point of the tutorial is more of a detour